includes/utils/class-amp-validation-utils.php

```diff
@@ -1791,4 +1792,22 @@ public static function get_recheck_link( $post, $redirect_url, $recheck_url = nu
 		);
 	}
 
+	/**
+	 * Outputs an admin notice if persistent object cache is not present.
+	 *
+	 * @return void
+	 */
+	public static function persistent_object_caching_notice() {
+		if ( ! wp_using_ext_object_cache() && 'toplevel_page_amp-options' === get_current_screen()->id ) {
+			printf(
+				'<div class="notice notice-warning is-dismissible"><p>%s <a href="%s">%s</a></p><button type="button" class="notice-dismiss"><span class="screen-reader-text">%s</span></button></div>',
+				esc_html__( 'The AMP plugin performs at its best when persistent object cache is enabled.', 'amp' ),
+				esc_url( 'https://codex.wordpress.org/Class_Reference/WP_Object_Cache#Persistent_Caching' ),
+				esc_html__( 'More details', 'amp' ),
+				esc_html__( 'Dismiss this notice.', 'amp' )
```

Comment:
I think we should eliminate the dismissal because as soon as the page reload the notice comes right back.

Reply:
@oscarssanchez once this change is made the PR will be good to merge.